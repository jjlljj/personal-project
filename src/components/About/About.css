.about {
  width: 100%;
}

.about h2 {
  text-align: center;
  font-size: 28px;
  margin: 25px;
}

.about .text-wrap {
  min-width: 300px;
  max-width: 600px;
  margin: 20px auto;
}

.about p {
  line-height: 24px;
  margin: 20px;
}

.about .btn-wrap {
  min-width: 300px;
  max-width: 600px;
  margin: 20px auto;
}

.about .btn-wrap button {
  background-color: #f3f3f3;
  border: none;
  box-sizing: border-box;
  display: inline-block;
  font-size: 14px;
  height: 44px;
  width: 100%;
}

.about .btn-wrap button:active {
  background: blue;
}

.about .btn-wrap a {
  width: 48%;
  margin: 20px;
}