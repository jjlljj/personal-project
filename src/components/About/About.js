import React from 'react'
import './About.css'

export const About = (props) => {

  return (
    <div className="about">
      <h2>Interpret, Don't Misinterpret</h2>
      <div className="text-wrap"> 
      <p>We negotiate every day. Whether by email, chat, instant message, we are constantly navigating new spaces in which the meaning of our words matter. These can be quick and off the cuff, or they can allow time for reflection and careful choice of language.
      </p>
      <p>
        Yet often, these lack the cues of tone, intent and meaning that we pick up when we hear and see the party we are negotiating with. Using e-communication technologies we sometimes send unintended messages or misinterpret a message we receive. As a result, rather than reaching a resolution through the negotiation, the conflict escalates.
      </p>
      </div>

      <h2>Tones:</h2>
      <div className="text-wrap"> 
        
      </div>
    </div>
  )
}

export default About