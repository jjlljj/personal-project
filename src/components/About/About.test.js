/* eslint-disable */
import React from 'react'
import { shallow } from 'enzyme'
import About from './About'

describe('About', () => {
  let renderedComponent

  beforeEach(() => {
    renderedComponent = shallow(<About />)
  })

  it('should match snapshot', () => {

    expect(renderedComponent).toMatchSnapshot()
  })
})
