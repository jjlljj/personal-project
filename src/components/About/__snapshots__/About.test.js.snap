// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          Interpret, Don't Misinterpret
</h2>,
        <div
          className="text-wrap"
>
          <p>
                    We negotiate every day. Whether by email, chat, instant message, we are constantly navigating new spaces in which the meaning of our words matter. These can be quick and off the cuff, or they can allow time for reflection and careful choice of language.
          </p>
          <p>
                    Yet often, these lack the cues of tone, intent and meaning that we pick up when we hear and see the party we are negotiating with. Using e-communication technologies we sometimes send unintended messages or misinterpret a message we receive. As a result, rather than reaching a resolution through the negotiation, the conflict escalates.
          </p>
</div>,
        <h2>
          Tones:
</h2>,
        <div
          className="text-wrap"
/>,
      ],
      "className": "about",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Interpret, Don't Misinterpret",
        },
        "ref": null,
        "rendered": "Interpret, Don't Misinterpret",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p>
              We negotiate every day. Whether by email, chat, instant message, we are constantly navigating new spaces in which the meaning of our words matter. These can be quick and off the cuff, or they can allow time for reflection and careful choice of language.
</p>,
            <p>
              Yet often, these lack the cues of tone, intent and meaning that we pick up when we hear and see the party we are negotiating with. Using e-communication technologies we sometimes send unintended messages or misinterpret a message we receive. As a result, rather than reaching a resolution through the negotiation, the conflict escalates.
</p>,
          ],
          "className": "text-wrap",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "We negotiate every day. Whether by email, chat, instant message, we are constantly navigating new spaces in which the meaning of our words matter. These can be quick and off the cuff, or they can allow time for reflection and careful choice of language.",
            },
            "ref": null,
            "rendered": "We negotiate every day. Whether by email, chat, instant message, we are constantly navigating new spaces in which the meaning of our words matter. These can be quick and off the cuff, or they can allow time for reflection and careful choice of language.",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Yet often, these lack the cues of tone, intent and meaning that we pick up when we hear and see the party we are negotiating with. Using e-communication technologies we sometimes send unintended messages or misinterpret a message we receive. As a result, rather than reaching a resolution through the negotiation, the conflict escalates.",
            },
            "ref": null,
            "rendered": "Yet often, these lack the cues of tone, intent and meaning that we pick up when we hear and see the party we are negotiating with. Using e-communication technologies we sometimes send unintended messages or misinterpret a message we receive. As a result, rather than reaching a resolution through the negotiation, the conflict escalates.",
            "type": "p",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Tones:",
        },
        "ref": null,
        "rendered": "Tones:",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "text-wrap",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            Interpret, Don't Misinterpret
</h2>,
          <div
            className="text-wrap"
>
            <p>
                        We negotiate every day. Whether by email, chat, instant message, we are constantly navigating new spaces in which the meaning of our words matter. These can be quick and off the cuff, or they can allow time for reflection and careful choice of language.
            </p>
            <p>
                        Yet often, these lack the cues of tone, intent and meaning that we pick up when we hear and see the party we are negotiating with. Using e-communication technologies we sometimes send unintended messages or misinterpret a message we receive. As a result, rather than reaching a resolution through the negotiation, the conflict escalates.
            </p>
</div>,
          <h2>
            Tones:
</h2>,
          <div
            className="text-wrap"
/>,
        ],
        "className": "about",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Interpret, Don't Misinterpret",
          },
          "ref": null,
          "rendered": "Interpret, Don't Misinterpret",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p>
                We negotiate every day. Whether by email, chat, instant message, we are constantly navigating new spaces in which the meaning of our words matter. These can be quick and off the cuff, or they can allow time for reflection and careful choice of language.
</p>,
              <p>
                Yet often, these lack the cues of tone, intent and meaning that we pick up when we hear and see the party we are negotiating with. Using e-communication technologies we sometimes send unintended messages or misinterpret a message we receive. As a result, rather than reaching a resolution through the negotiation, the conflict escalates.
</p>,
            ],
            "className": "text-wrap",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "We negotiate every day. Whether by email, chat, instant message, we are constantly navigating new spaces in which the meaning of our words matter. These can be quick and off the cuff, or they can allow time for reflection and careful choice of language.",
              },
              "ref": null,
              "rendered": "We negotiate every day. Whether by email, chat, instant message, we are constantly navigating new spaces in which the meaning of our words matter. These can be quick and off the cuff, or they can allow time for reflection and careful choice of language.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Yet often, these lack the cues of tone, intent and meaning that we pick up when we hear and see the party we are negotiating with. Using e-communication technologies we sometimes send unintended messages or misinterpret a message we receive. As a result, rather than reaching a resolution through the negotiation, the conflict escalates.",
              },
              "ref": null,
              "rendered": "Yet often, these lack the cues of tone, intent and meaning that we pick up when we hear and see the party we are negotiating with. Using e-communication technologies we sometimes send unintended messages or misinterpret a message we receive. As a result, rather than reaching a resolution through the negotiation, the conflict escalates.",
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Tones:",
          },
          "ref": null,
          "rendered": "Tones:",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "text-wrap",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
